src/main/resources/hudson/tasks/Mailer/config_es.properties

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-description=Lista de destinatarios separadas por un espacio en blanco. El correo será enviado siempre que una ejecución falle.
+description=Lista de destinatarios separadas por un espacio en blanco. El correo ser\u00e1 enviado siempre que una ejecuci\u00f3n falle.
```

Comment:
@fcojfernandez FYI. I was able to open these files in NetBeans before the patch (mojibake) and after (looks like Spanish to me).

Reply:
i.e. this is really

```diff
-description=Lista de destinatarios separadas por un espacio en blanco. El correo serÃ¡ enviado siempre que una ejecuciÃ³n falle.
+description=Lista de destinatarios separadas por un espacio en blanco. El correo será enviado siempre que una ejecución falle.
```